includes/utilities.inc

```diff
@@ -447,7 +480,26 @@ function islandora_solr_islandora_basic_collection_backend_callable($collection_
  *   The updated display values.
  */
 function islandora_solr_truncate_field_display($display_values, $max_length, $add_ellipsis, $word_safe, $wordsafe_length, $separator, $link_options = NULL) {
+  $allowed_tags = array(
+    'a',
+    'em',
+    'strong',
+    'cite',
+    'blockquote',
+    'code',
+    'ul',
+    'ol',
+    'li',
+    'dl',
+    'dt',
+    'dd',
+    'br',
+    'span',
+    'div',
+  );
```

Comment:
So because constant arrays weren't a thing until PHP 5.6, I create an islandora_solr_get_filter_tags function instead which returns the arrays. Hope this is an acceptable solution. Thanks for the feedback/review!